probe-rs/targets/SAM D Series.yaml

```diff
@@ -0,0 +1,540 @@
+---
```

Comment:
right, i just used the default name from the generator tool. do you have documentation for how you would like the file names specified?

Reply:
The generated one is the correct one :) Please don't alter it :) Because if we regenerate we have to keep in mind to change it. If we just leave it be, it will automatically be replaced correctly :)